.buildkite/steps/copy.sh

```diff
@@ -110,15 +112,16 @@ if [[ $BUILDKITE_BRANCH != "master" ]] && [[ "$BUILDKITE_TAG" != "$BUILDKITE_BRA
   cat << EOF > "$mapping_file"
 Mappings:
   AWSRegion2AMI:
-    ${AWS_REGION} : { linux: $linux_source_image_id, windows: $windows_source_image_id }
+    ${AWS_REGION} : { linux: $linux_amd64_source_image_id, linuxarm: $linux_arm64_source_image_id, windows: $windows_amd64_source_image_id }
```

Comment:
`linuxarm`… should we use something like `linux-arm64` or `linux_arm64`?
Or is this referencing something outside our control?

Reply:
These map keys are totally arbitrary and can be whatever we want. I'll update them.